apps/web/src/utils/helpers.ts

```diff
@@ -269,3 +269,10 @@ export function unpackOptionalArray<T = []>(
   }
   return array
 }
+
+export function maxChar(str: string, maxLength: number) {
```

Comment:
The goal of using maxChar is that I can lop off a portion of the string and still display the unit 'Eth', which I won't be able to do without some really crazy CSS. 

Also, placeholder-shown is a class for targeting an input or textarea only when the placeholder is shown. Don't think it would solve the same problem. 

https://developer.mozilla.org/en-US/docs/Web/CSS/:placeholder-shown

 Likewise, using targeting ::placeholder  doesn't really solve the problem either. 
 
 This
 ```
 '::placeholder': {
    textOverflow: 'ellipsis',
    overflow: 'hidden',
    whiteSpace: 'nowrap'
  }
 ```
 results in this 
  ![image](https://github.com/ourzora/nouns-builder/assets/53406838/64c4b8cc-8540-491f-b181-fb2ce4167d82)
 
 Which, IMO is less desirable.